### ISSUE TITLE
Accessibility - elements not labeled

### ISSUE THREAD

**Issue by heull001** [nucleus-ffm/foss_warn#175]
In the locations tab, the button with more information about the location and the tick which probably indicates that notifications are activated, are not labeled and so not correctly spoken by screenreaders. Additionally the tick is identified as button, but does not seem to be one.
